<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTViewer</class>
 <widget class="QWidget" name="CTViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CT Viewer</string>
  </property>

  <!-- Grid Layout for the axial, coronal, sagittal, and 3D view -->
  <layout class="QGridLayout" name="gridLayout">
   <!-- Axial View -->
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="axialLayout">
     <item>
      <widget class="QLabel" name="axialLabel">
       <property name="text">
        <string>Axial View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="vtkWidget_axial"/> <!-- Placeholder -->
     </item>
     <item>
      <widget class="QSlider" name="slider_axial">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- Coronal View -->
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="coronalLayout">
     <item>
      <widget class="QLabel" name="coronalLabel">
       <property name="text">
        <string>Coronal View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="vtkWidget_coronal"/> <!-- Placeholder -->
     </item>
     <item>
      <widget class="QSlider" name="slider_coronal">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- Sagittal View -->
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="sagittalLayout">
     <item>
      <widget class="QLabel" name="sagittalLabel">
       <property name="text">
        <string>Sagittal View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="vtkWidget_sagittal"/> <!-- Placeholder -->
     </item>
     <item>
      <widget class="QSlider" name="slider_sagittal">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- 3D Model -->
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="modelLayout">
     <item>
      <widget class="QLabel" name="modelLabel">
       <property name="text">
        <string>3D Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="model_vtkWidget"/> <!-- Placeholder -->
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
